internal/licenseutils/utils.go

```diff
@@ -97,7 +97,7 @@ func (u HubUser) GetAvailableLicenses(ctx context.Context) ([]LicenseDisplay, er
 	// Filter out expired licenses
 	i := 0
 	for _, s := range subs {
-		if s.State != "expired" && s.Expires != nil {
+		if s.State == "active" && s.Expires != nil {
```

Comment:
What are the different states? I think these constants should be extracted and exposed and documented as a type in the licensing library, instead of plain string like this.

Reply:
Good call, I opened an issue in docker/licensing to address this. Can swing back to this after that change and vendor bump happen.